recipe/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pycountry" %}
+{% set version = "24.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b61b3faccea67f87d10c1f2b0fc0be714409e8fcdcc1315613174f6466c10221
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry-core
```

https://github.com/pycountry/pycountry/blob/24.6.1/pyproject.toml#L70
```suggestion
    - poetry-core >=1.0.0
```